Tests/Pcap++Test/Tests/IpMacTests.cpp

```diff
@@ -1,18 +1,17 @@
-#include "../TestDefinition.h"
-#include "../Common/TestUtils.h"
 #include "../Common/GlobalTestArgs.h"
-#include <sstream>
-#include <algorithm>
+#include "../Common/TestUtils.h"
+#include "../TestDefinition.h"
 #include "EndianPortable.h"
-#include "Logger.h"
 #include "GeneralUtils.h"
 #include "IpAddress.h"
-#include "MacAddress.h"
 #include "LRUList.h"
+#include "Logger.h"
+#include "MacAddress.h"
 #include "NetworkUtils.h"
 #include "PcapLiveDeviceList.h"
 #include "SystemUtils.h"
-
+#include <algorithm>
+#include <sstream>
```

Comment:
any reason why you changed the include order?

Reply:
Sorry, this seemed to be a problem with my auto-formatter that kicked in. I will try to avoid using the auto-formatter when working on PcapPlusPlus code from now on to avoid any conflicts with "blame" and similar.